rts/lra/client/src/main/java/io/narayana/lra/client/internal/NarayanaLRAClient.java

```diff
@@ -449,11 +449,13 @@ public void leaveLRA(URI lraId, String body) throws WebApplicationException {
     }
 
     /**
-     * For particular compensator class it returns termination uris based on the provided base uri.
-     * You get map of string and URI.
+     * For particular compensator class it returns termination uris based on the
+     * provided base uri. You get map of string and URI.
      *
-     * @param compensatorClass  compensator class to examine
-     * @param uriInfo  the uri that triggered this join request.
+     * @param compensatorClass compensator class to examine.
+     * @param uriPrefix        the uri that triggered this join request.
```

Comment:
This commit works for me, thank you. But I added a couple of other comments that to look at and follow up on before I approve the pull request.

Reply:
Sorry, I have subsequently asked about the reformatting above.